contrib/ansible/README.md

```diff
@@ -1,5 +1,7 @@
 # Build a Kubernetes cluster using k3s via Ansible.
 
+https://github.com/itwars
```

Comment:
What's this link for?  Do you want to add "Author: " or something to just indicate what this link is.

Reply:
Yes it's Author: